mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -129,7 +124,8 @@
 <aui:script use="todo">
 	new Liferay.Portlet.Todo(
 		{
-			namespace: '<portlet:namespace />'
+			namespace: '<portlet:namespace />',
+			taskFinished: '<%= UnicodeLanguageUtil.get(pageContext, "task-finished") %>'
```

Comment:
You still did not include ```Language.properties``` file, so localization still does not work. 

I see you sent it in a separate PR a day later, but one PR should be one functional whole, unless you have a very good reason why you are sending only a part, like too much code. Language.properties does not fall into that category, as it does not contain any code.